wtr-utils.js

```diff
@@ -294,9 +294,8 @@ const createVisualTestsConfig = (theme) => {
         getBaselineName: getBaselineScreenshotName,
         getDiffName: getDiffScreenshotName,
         getFailedName: getFailedScreenshotName,
-        diffOptions: {
-          threshold: 0.2,
-        },
```

Comment:
As we previously figured out, this threshold didn't work properly (which could be seen in failed builds).
Updated to use better checks added in https://github.com/modernweb-dev/web/pull/1879